tests/installation/setup_zdup.pm

```diff
@@ -19,8 +19,8 @@ use utils;
 sub run() {
     my ($self) = @_;
 
-    $self->wait_boot(ready_time => 600);
-
+    assert_screen 'displaymanager', 500;
+    ensure_unlocked_desktop;
```

Comment:
you know nothing, this is not exchange, try it by your self and then write something sane